pkg/doc/signature/proof/jws.go

```diff
@@ -143,71 +143,9 @@ func prepareDocumentForJWS(suite signatureSuite, jsonldObject map[string]interfa
 
 func getCompactedWithSecuritySchema(docMap map[string]interface{},
 	opts ...jsonld.ProcessorOpts) (map[string]interface{}, error) {
-	var contextMap map[string]interface{}
-
-	err := json.Unmarshal([]byte(securityJSONLD), &contextMap)
-	if err != nil {
-		return nil, err
+	contextMap := map[string]interface{}{
+		"@context": "https://w3id.org/security/v2",
 	}
 
 	return jsonld.Default().Compact(docMap, contextMap, opts...)
 }
-
-// cached value from https://w3id.org/security/v2
```

Comment:
whats the impact of not using cache here?

if internet fetch is needed, we should be preloading.

Reply:
I agree. We should preload `https://w3id.org/security/v2` (and dependent https://w3id.org/security/v1 as well) the same way we pre-load `https://www.w3.org/2018/credentials/v1` for VC/VP.

Reply:
I created a separate issue for that (#2253) in order to not overload current PR.